logstash-filter-fieldmap.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'logstash-filter-fieldmap'
-  s.version         = '0.0.3'
+  s.version         = '0.0.4'
```

Comment:
Probably worth bumping the minor rev number. This seems like more than just a patch :-)